omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/PersonResource1_10.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_10;
+
+import io.swagger.models.Model;
+import org.openmrs.Person;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.annotation.Resource;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingCrudResource;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.PersonResource1_8;
+
+/**
+ * {@link Resource} for Person, supporting standard CRUD operations
+ */
+@Resource(name = RestConstants.VERSION_1 + "/person", order = 3, supportedClass = Person.class, supportedOpenmrsVersions = {
+        "1.10.4 - 1.10.6" })
+public class PersonResource1_10 extends PersonResource1_8 {
+
+    /**
+     * @see DelegatingCrudResource#getRepresentationDescription(Representation)
+     */
+    @Override
+    public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
+        DelegatingResourceDescription description = super.getRepresentationDescription(rep);
+        if (description != null) {
+            description.addProperty("birthtime");
```

Comment:
@Andu033 it just occurred to me that `PersonResource1_11` should probably now extend `PersonResource1_10`, and thereby stop adding the property `"birthtime"`, see [here](https://github.com/openmrs/openmrs-module-webservices.rest/blob/527db5acb23147a6e4fc6001cb4664037b116a1c/omod-1.11/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_11/PersonResource1_11.java#L37-L68).

Reply:
Birthtime was also backported to 1.11 too, I think in 1.11.4, therefore PersonResource1_11 can't only just extend PersonResource1_10, it would also need to have some smarter logic to exclude the property by checking the openmrs version one is running.

Reply:
Correct, `birthtime` appears from 1.11.4...

@wluyima are we going to have a `PersonResource1_11` and `PersonResource1_11_4`??

Reply:
You could add smart logic in PersonResource1_11 that includes the property depending on the version it's running on